provider/datagen/src/lib.rs

```diff
@@ -298,13 +291,10 @@ fn test_keys() {
 }
 
 #[test]
-fn test_keys_from_file() {
-    let file = std::fs::File::open(
-        PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("tests/testdata/work_log+keys.txt"),
-    )
-    .unwrap();
+fn test_keys_from_bin() {
     assert_eq!(
-        keys_from_file(file).unwrap(),
+        keys_from_bin(PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("tests/data/work_log.wasm"))
```

Comment:
Nit: It looks like *work_log.wasm* is a one-off file that does not need an automated job to re-generate it. Please:

1. Re-build it with empty data so that it isn't 2.47 MB
2. Leave some instructions on how to manually re-create the file if we ever want to do so in the future

Reply:
nit: also, can we call it work_log_keys.wasm or something?

Reply:
probably add a regenerate-work-log.wasm or something